winpr/libwinpr/utils/lodepng/lodepng.c

```diff
@@ -1151,13 +1151,7 @@ static unsigned inflateHuffmanBlock(ucvector* out, const unsigned char* in, size
       code_d = huffmanDecodeSymbol(in, bp, &tree_d, inbitlength);
       if(code_d > 29)
       {
-        if(code_ll == (unsigned)(-1)) /*huffmanDecodeSymbol returns (unsigned)(-1) in case of error*/
```

Comment:
behavioural change, justification?

Reply:
Always false
We have a condition for this branch above:
code_ll <= LAST_LENGTH_CODE_INDEX (285)
So, code_ll never been equal to 4294967295
Maybe LAST_LENGTH_CODE_INDEX will change, but hardly so much

Reply:
well, in this case I´d stick on the safe side and keep it

Reply:
Ok, returned back